src/components/HistoryRecord/HistoryRecord.module.css

```diff
@@ -0,0 +1,108 @@
+.HistoryRecordWrapper {
```

Comment:
We have new HistoryRecord. Do we really need to add so many overrides. https://taskany-inc.github.io/bricks/?path=/story/harmony-historyrecord--default